src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -81,6 +83,14 @@ const OrgProfileTab: FC = () => {
   const allowSelfRemoval = users.length > 1
   const showLeaveOrgButton = isFlagEnabled('createDeleteOrgs')
   const hasFetchedOrgDetails = orgDetailsStatus === RemoteDataState.Done
+  const hasFetchedStorageType = Boolean(storageType)
+
+  const storageTypeMap = {
+    iox: 'IOx',
+    tsm: 'TSM',
+  }
+
+  const formattedStorageType = storageTypeMap[storageType] || storageType
 
```

Comment:
I removed the `toLowerCase()` here. It isn't necessary since the endpoint is already sending back a lowercase response. It also causes an error in OSS, where storageType isn't returned (can't run the `toLowerCase` method on an undefined, and not worth overcomplicating this by adding in additional logic to check whether the type is `string` or `undefined`).